glue/viewers/table/qt/tests/test_data_viewer.py

```diff
@@ -460,6 +461,52 @@ def test_incompatible_subset():
     assert refresh2.call_count == 0
 
 
+@requires_pyqt_gt_59
```

Comment:
```suggestion
@requires_pyqt_gt_59_or_pyside2
```
Getting lengthy, but probably better to stay verbose... ;-)